README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
- [ ] Please make the comments polite, and easy to understand and give reasons why you are suggesting a change. Currently, your comments sound like a command, they are vague and hard to understand and there are no reasons why the changes were suggested. Using a polite tone makes communication smoother between the author and the reviewer. 😄📗 

 #### Example of Polite comments 
 
> **_Kindly_** use `rem` or `em` units instead of `px`. 
> I think it would be better to use `rem` or `em` units instead of `px`. 

#### Example of a clear and easy-to-understand comment that gives reason

 > **_Kindly_** use Stylelint linters for this project. Stylelint reports bad codes in your CSS files. It helps to enforce consistent code and prevents you from making errors in your stylesheets. 👍 😎 
 
 > Kindly put the nav tag into the header tag. The nav tag is a semantic tag that should contain the logo and other navigational links to parts of the site or external sites. 
 
 >The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?
 
 #### Hint: You don't have to create another inline comment. You could edit the inline comments that you have above. 💻